.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -1,23 +1,26 @@
 name: Feature request
 description: Suggest an idea for this project
-labels: ["feature"]
+labels: ["feature,needs-triage"]
 body:
   - type: textarea
     attributes:
-      label: Is your feature request related to a problem? Please describe.
+      label: What
```

Comment:
Shouldn't it be "Why"?

Reply:
The current structure is What for problem description. We can add a Why section as well

Reply:
Not too much section (it becomes cumbersome), I think it's ok like that. Maybe in placeholder we could recall to give some context if needed.